src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -13,78 +12,59 @@ import { MenuItem } from '../../../../components/menu';
 type AnnotationActivityMenuProps = {
     canDelete?: boolean,
     canEdit?: boolean,
+    className?: string,
     id: string,
-    onDeleteConfirm: () => void,
+    isDisabled?: boolean,
+    onDelete: () => void,
     onEdit: () => void,
+    onMenuClose: () => void,
+    onMenuOpen: () => void,
 };
 
-const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdit }: AnnotationActivityMenuProps) => {
-    const [isConfirmingDelete, setIsConfirmingDelete] = React.useState(false);
-
-    const handleDeleteCancel = (): void => {
-        setIsConfirmingDelete(false);
-    };
-
-    const handleDeleteClick = () => {
-        setIsConfirmingDelete(true);
-    };
-
-    const handleDeleteConfirm = () => {
-        setIsConfirmingDelete(false);
-        onDeleteConfirm();
-    };
-
-    const tetherProps = {
-        attachment: 'top right',
-        className: 'bcs-AnnotationActivity-deleteConfirmationModal',
-        constraints: [{ to: 'scrollParent', attachment: 'together' }],
-        targetAttachment: 'bottom right',
-    };
-
-    return (
-        <TetherComponent {...tetherProps}>
-            <Media.Menu
-                isDisabled={isConfirmingDelete}
-                data-testid="annotation-activity-actions-menu"
-                menuProps={{
-                    'data-resin-component': 'preview',
-                    'data-resin-feature': 'annotations',
-                }}
+const AnnotationActivityMenu = ({
+    canDelete,
+    canEdit,
+    className,
+    id,
+    isDisabled,
+    onDelete,
+    onEdit,
+    onMenuClose,
+    onMenuOpen,
+}: AnnotationActivityMenuProps) => (
+    <Media.Menu
```

Comment:
Should this still be a `Media.Menu` since it's been moved out of its `Media` parent?